packages/aws-cdk/test/test.cdk-toolkit.ts

```diff
@@ -22,6 +22,27 @@ export = nodeunit.testCase({
         // WHEN
         toolkit.deploy({ stackNames: ['Test-Stack-A', 'Test-Stack-B'], sdk: new SDK() });
 
+        // THEN
+        test.done();
+      },
+      'with sns notification arns'(test: nodeunit.Test) {
+        // GIVEN
+        const notificationArns = ['arn:aws:sns:::cfn-notifications', 'arn:aws:sns:::my-cool-topic'];
+        const toolkit = new CdkToolkit({
+          appStacks: new TestAppStacks(test),
+          provisioner: new TestProvisioner(test, {
+            'Test-Stack-A': { Foo: 'Bar' },
+            'Test-Stack-B': { Baz: 'Zinga!' },
+          }, notificationArns),
+        });
+
+        // WHEN
+        toolkit.deploy({
+          stackNames: ['Test-Stack-A', 'Test-Stack-B'],
+          notificationArns,
+          sdk: new SDK()
+        });
```

Comment:
Where are the assertions? I'd expect some expectation to be asserted...

Reply:
they're in the deploy method of this itself. In the deploy method of this Test, the tags and notification arns are being verified.
```
this.test.deepEqual(options.notificationArns, this.expectedNotificationArns);
```

I extended it based on how it was previously configured, but maybe it should be more intuitive.